cpp/open3d/visualization/visualizer/O3DVisualizer.cpp

```diff
@@ -1706,6 +1711,14 @@ Ctrl-alt-click to polygon select)";
         }
     }
 
+    void SetCollapsableVertOpen(const std::string &name, bool open) {
+        auto it = side_panel_verts_.find(name);
+        if (it != side_panel_verts_.end() && it->second) {
+            it->second->SetIsOpen(open);
+            window_->SetNeedsLayout();
+        }
```

Comment:
I think we don't need `side_panel_verts_` and can remove it. Instead an if-else works:
```cpp
if (name=='Geometries') {
settings.geometries_panel.SetIsOpen(open);
...
} else if (name==...) {
...
} else if ...
```
Code wise it's about the same, all the logic is collected in this one function (no updates to `side_panel_verts_` needed.

Reply:
Updated in commit https://github.com/isl-org/Open3D/pull/6865/commits/a19cdf12bc6329a88047f2c602175005f7267eaf